docs/source/how_tos/set_up_standalone_registry.rst

```diff
@@ -0,0 +1,154 @@
+.. _set_up_standalone_registry:
+
+***********************************
+How to Set Up a Standalone Registry
+***********************************
+
+dsgrid stores registry information in an ArangoDB database. If you want to work with a completely 
+local version of dsgrid, you must set up and connect to a local ArangoDB instance. There are two 
+ways to install ArangoDB on your computer: `Native Installation`_ or `Docker Container`_. You might 
+also want to set up a standalone registry on `NREL HPC`_ for development or testing purposes.
+
+Once installed, the easiest way to mange the database manually is through Arango's web UI,
+available at http://localhost:8529.
+
+Native Installation
+===================
+
+1. Install **ArangoDB Community Edition** locally by following the instructions at
+   https://www.arangodb.com/download-major/. If asked and you plan to run dsgrid tests, set the root 
+   password to openSesame to match the defaults.
+
+   Add the ``bin`` directory to your system path and customize the configuration files, 
+   particularly regarding authentication, as desired. 
+
+    .. tabs::
+
+        .. tab:: Mac
```

Comment:
Arango is now making it extremely difficult to install their software on Mac. Docker is the only real solution going forward. We’ll need to update this text - but not in this PR. I’ll take care of it later.